### PR TITLE
individual scalatest in bazel md

### PR DESCRIPTION
**Pull request by dasormeter**
Include instructions to run specific test in bazel

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: BAZEL.md [hunk 1]
```diff
@@ -361,17 +361,24 @@ detailed information.
...
 
     ```
-    bazel test //ledger-client/ods:tests_test_suite_src_test_scala_com_digitalasset_ods_slick_SqlUtilsTest.scala
+    bazel test //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala
     ```
+    
+- Execute a test with a specific name
+
+    ```
+    bazel test//ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala --test_arg=-t "JDBC Ledger DAO should be able to persist and load contracts without external offset"
```
Comment (SamirTalwar) [inline BAZEL.md]: Don't you need two `--test-arg` switches here?

Comment (dasormeter) [reply BAZEL.md]: indeed! now corrected

File: BAZEL.md [hunk 2]
```diff
@@ -361,17 +361,24 @@ detailed information.
...
 
     ```
-    bazel test //ledger-client/ods:tests_test_suite_src_test_scala_com_digitalasset_ods_slick_SqlUtilsTest.scala
+    bazel test //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala
     ```
+    
+- Execute a test with a specific name
+
+    ```
+    bazel test//ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala --test_arg=-t --test_arg="JDBC Ledger DAO should be able to persist and load contracts without external offset"
```
Comment (SamirTalwar) [inline BAZEL.md]: Another typo: you need a space after `bazel test`.

File: BAZEL.md [hunk 3]
```diff
@@ -361,17 +361,24 @@ detailed information.
...
+    ```    
 
 - Pass an argument to a test case in a Scala test-suite
 
     ```
     bazel test //ledger-client/ods:tests_test_suite_src_test_scala_com_digitalasset_ods_slick_SqlUtilsTest.scala --test_arg=-z --test_arg="should return true"
     ```
+    More broadly, for scala tests you can pass through any of the args outlined in scalatest document, separating into two instances of the --test-arg parameter as shown in the two examples above.  See http://www.scalatest.org/user_guide/using_the_runner for a full list of args you can pass through.
```
Comment (SamirTalwar) [inline BAZEL.md]: Perhaps wrap `--test-arg` in backticks.

Comment (SamirTalwar) [reply BAZEL.md]: Capitalize "Scala" and "ScalaTest"?

**Pull request merged**